stylus/ui-components/popover.styl

```diff
@@ -27,6 +27,7 @@ $popover
         border-top  1px solid grey-09
 
     a
+    [role=button]
```

Comment:
Could you just add the element `button` as a selector too so every case is covered, please?